weblate/templates/snippets/progress.html

```diff
@@ -2,12 +2,9 @@
 
 <div class="progress">
```

Comment:
Include the title from the last part of the progress bar which got removed:

```suggestion
<div class="progress" title="{% trans "Needs attention" %}">
```

Alternatively, keep the last part just to show the tooltip.